iapetus/membraneModeller.py

```diff
@@ -0,0 +1,144 @@
+# ==============================================================================
+# FILE DOCSTRING
+# ==============================================================================
+
+"""
+
+Membrane Modeller
+=================
+
+Add tools to the OpenMM Modeller_ to make MemProtMD_ coarse-grained membranes
+compatibles with the all-atom AMBER force field.
+
+.. _Modeller: http://docs.openmm.org/latest/api-python/generated/simtk.openmm.app.modeller.Modeller.html#simtk.openmm.app.modeller.Modeller
+.. _MemProtMD: http://memprotmd.bioch.ox.ac.uk/home/
+
+
+"""
+
+import os
+import simtk.openmm.app as app
+
+class Modeller(app.Modeller):
```

Comment:
I'd call this `MembraneModeller` to avoid confusion of names with `app.Modeller`.